extra/opensslpp/src/opensslpp/evp_pkey.cpp

```diff
@@ -20,6 +20,18 @@
 
 #include <openssl/evp.h>
 #include <openssl/pem.h>
+#include <openssl/rsa.h>  // Needed for EVP_PKEY_get0_RSA
+
+#ifndef EVP_PKEY_get0_RSA
+extern "C" {
+static inline const RSA *EVP_PKEY_get0_RSA(const EVP_PKEY *pkey) {
```

Comment:
This is hard to verify as implementation is different in various OpenSSL versions but probably a client doesn't use
`Enterprise Encryption UDFs for OpenSSL` at all so we should be safe.

Reply:
I just tried to adopt it to solve compilation issue with openssl1.0.2k and as this build should be used just for tests I hope it should be fine :)